cmd/satellite/entrypoint

```diff
@@ -25,4 +25,8 @@ fi
 
 RUN_PARAMS="${RUN_PARAMS:-} --config-dir ${CONF_PATH}"
 
+if [ "${SATELLITE_API:-}" = true ]; then
```

Comment:
`true` isn't anything special in bash, it's just a string like `"true"`. It could be argued that this is misleading to those familiar with other languages, as `true` is also a valid command. I'd recommend putting quotes around it to help those confusions.

Reply:
sounds good.